libs/components/indicators/src/lib/modules/status-indicator/status-indicator.component.scss

```diff
@@ -31,6 +31,18 @@
 }
 
 @include mixins.sky-theme-modern {
+  .sky-status-indicator {
+    --sky-status-indicator-icon-inverse-color: #{$sky-text-color-default};
+  }
+
+  .sky-status-indicator-icon {
```

Comment:
specify class as `sky-status-indicator-icon` and not just `sky-status-indicator` because status indicator supports inline-help which also uses `sky-icon`.